From 5cf67ac4117f14fab8fdd72ed41f0656e8533382 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <vbarshak@dev.rtsoft.ru>
Date: Tue, 23 Apr 2013 16:17:51 +0400
Subject: [PATCH 073/127] arm: tegra: ktt30: Enable KXTF9 accelerometer input
 driver

This disables the Invensense MPU drivers and enables preliminary
MPU KXTF9 input driver support in ktt30_defconfig.

Signed-off-by: Valentine Barshak <vbarshak@dev.rtsoft.ru>
---
 arch/arm/configs/ktt30_defconfig |    7 +++----
 1 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/arch/arm/configs/ktt30_defconfig b/arch/arm/configs/ktt30_defconfig
index 2a12c4c..0a65c4c 100644
--- a/arch/arm/configs/ktt30_defconfig
+++ b/arch/arm/configs/ktt30_defconfig
@@ -201,10 +201,7 @@ CONFIG_UID_STAT=y
 CONFIG_BCM4329_RFKILL=y
 CONFIG_TEGRA_CRYPTO_DEV=y
 CONFIG_MPU_SENSORS_TIMERIRQ=y
-CONFIG_MPU_SENSORS_MPU3050=m
-CONFIG_MPU_SENSORS_KXTF9=m
-CONFIG_MPU_SENSORS_AK8963=m
-CONFIG_MPU_SENSORS_AK8975=m
+# CONFIG_INV_SENSORS is not set
 CONFIG_TEGRA_BB_SUPPORT=y
 CONFIG_TEGRA_BB_POWER=y
 CONFIG_TEGRA_BB_M7400=y
@@ -260,6 +257,8 @@ CONFIG_JOYSTICK_XPAD=y
 CONFIG_INPUT_TOUCHSCREEN=y
 CONFIG_TOUCHSCREEN_ATMEL_MXT=y
 CONFIG_INPUT_MISC=y
+CONFIG_INPUT_MPU3050=m
+CONFIG_INPUT_KXTF9=m
 CONFIG_INPUT_UINPUT=y
 CONFIG_INPUT_GPIO=y
 CONFIG_INPUT_ALPS_GPIO_SCROLLWHEEL=y
-- 
1.7.4.4

